config/routes.rb

```diff
@@ -23,6 +23,9 @@
 
   resources :user, only: [:index]
   namespace :user do
+    resources :dashboard, only: %i[index]
+    get 'documents/:type/:id', to: 'documents#show', as: :document
+    post 'documents/:type/:id/remove', to: 'documents#remove', as: :document_remove
```

Comment:
There's gotta be a tidier way to do this 😬

Reply:
@Br3nda is making some other changes to the routes in #172 so I'm gonna say that we should have a wholesale rethink of them once both of these are merged